samples/snippets/snippets_test.py

```diff
@@ -189,6 +191,12 @@ def test_upload_blob(test_bucket):
         )
 
 
+def test_upload_blob_from_memory(test_bucket):
+    storage_upload_from_memory.upload_blob_from_memory(
+        test_bucket.name, "Hello, is it me you're looking for?", "test_upload_blob"
+    )
+
```

Comment:
You could either assert that the blob now exists or assert capsys.readouterr() here just to verify that the print statement successfully ran.